### DISCUSSION TITLE
Crowdin subscription expires soon

### DISCUSSION THREAD

**Discussion by rinsuki** [cinderella-project/iMast#195] [General]
iMast は Crowdin の bronze plan を翻訳管理に利用しています。

このプランは GitHub Student Developer Pack で1年間のクーポンを貰って使っているのですが、これが 2020/12/30 で終了します。また bronze plan (150USD/月) に支払えるお金を持っていない(し iMast の規模から見ると高すぎる)ため bronze plan の継続は不可能なので、別の手段を考える必要があります。

とりあえずのところ3つのプランを考えています:

- Crowdin の OSS Project Support を使う https://crowdin.com/page/open-source-project-setup-request
  - ただし条件に合致するかは不明 (News feed を最新に保ってって言われてるけどそもそも News feed なんてないし、企業名義で App Store に出してるとかで厳しいかも)
- 他の翻訳プラットフォームを使う (どこがいいの？)
- GitHub の Pull requests を翻訳に使う
  - 翻訳者がGitHub上でコントリビューターになるので草がより生えて嬉しいかも

Q. なんで bronze plan が必要なの？
A. GitHub 連携が bronze plan かそれ以降じゃないと使えないから

---

in iMast's Crowdin, currently using Crowdin's bronze plan (by GitHub Student Developer Pack coupon) with 1 year expiration.

this expire day is 2020/12/30, and I can't have a payable money (it is $150/mo) to bronze plan, so we should think next plan.

I have three plans:

- Use Crowdin's OSS Project Support https://crowdin.com/page/open-source-project-setup-request
  - but may we need some changes to website (crowdin requires  website of "news" section up-to-date, but iMast's website doesn't have a "news" section)
- Use other translation platform
- Use GitHub Pull requests to translate
  - this makes translators as GitHub's contributor, and translators GitHub grasses are more thick.

Q. why need bronze plan?
A. because iMast using GitHub Integration, and that requires bronze plan or more higher plan.

**Comment by wakest** [comment]
the first option seems like the simplest, making a news section where the news is this post would be sort of funny and also probably suitable...

**Comment by acid-chicken** [comment]
Crowdin for OSS
--

criteria は割と建前でそこまで厳密性は求められない印象（なんか 4 年ぐらい半凍結状態の自分のプロジェクトも未だに生きてる）。

Other platform
--

- [Weblate](https://weblate.org) が一見有料そうに見えるが、無料でセルフホスト可能。ちなみに GitHub の草もちゃんと生える。
- [GitLocalize](https://gitlocalize.com) は少し前に無料化されたが、クオリティはちょっと微妙（個人的な感想）。
- [POEditor](https://poeditor.com) も無料らしいが、自分は触ったことはない。

**Comment by rinsuki** [comment]
I'm finally bestir myself and start to decide translation platform.

I'm tried to Weblate, but self-hosting edition is eating 2GB RAM and that is not acceptable on my servers. then also tried to Weblate Hosting version, but every operation is slow (maybe speed of light is too slow?). so I decided to submit Crowdin's OSS Project Support, and crowdin accepted speedly, its 5 minutes 🤩

We can continute translate with https://crowdin.com/project/imast (2021/11/01: fix link), sorry for delay.

**Reply by comradekingu** [reply]
@rinsuki Did you set up shop on Weblate again?

**Reply by rinsuki** [reply]
@comradekingu nop, but I realized that my crowdin link are wrong. actual URL is this: https://crowdin.com/project/imast sorry for that!

**Reply by comradekingu** [reply]
I don't use that, and neither does anyone else.
The terms and conditions are horrendous, the functionality is bad, and the resulting quality is lackluster.
